scripts/buildAndroidSnapshots.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+
+/* eslint-disable no-console */
+
+const urlExists = require('url-exists');
+const child_process = require('child_process');
+
+const exec = (command, options) =>
+  child_process.execSync(command, {
+    stdio: 'inherit',
+    ...options,
+  });
+
+const DEFAULT_URL =
+  'http://trinerdis.cz:8000/repository/snapshots/com/trinerdis/skypicker/';
+
+if (!process.env.ANDROID_DEPLOYMENT_PASSWORD) {
+  throw Error('You need to pass ANDROID_DEPLOYMENT_PASSWORD.');
+}
+
+// Read native dependencies versions from package.json (maps, vector icons...)
+const buildPackage = require('../.build/package.json');
```

Comment:
This file is going to be eventually removed. Is there a better solution? (maybe just the `packing.json` from the root)

Reply:
Mmmm, if we cannot use that file, we will need to check several ones because you have dependencies spread. Like `react-native` in the root but others are in shared. Want me to change that?

Reply:
We went from this:

1. add native dep
2. add native dep to the package.json

To this:

1. add native dep
2. add native dep to the package.json
3. add native dep to the buildAndroidSnapshots.js

Instead of this:

1. add native dep

That's what I don't like about it - it's error prone.

Reply:
Mmm that is not entirely true because when you added a native dep, someone had to go here https://gitlab.skypicker.com/mobile/react-native-hotels and it + create the Snapshot process.

Now that we have it though all in one repo, maybe we can think of way to reduce steps.